py/picca/fitter2/pk.py

```diff
@@ -423,13 +423,15 @@ def pk_hcd_Rogers2018_uv_cross(k, pk_lin, tracer1, tracer2, **kwargs):
 def pk_gauss_smoothing(k, pk_lin, tracer1, tracer2, **kwargs):
     """
     Apply a Gaussian smoothing to the full correlation function
-
+    Args : k, sigma_smooth par and per
```

Comment:
This docstring does not seem right. It says "k, sigma_smooth", but the arguments are "k, pk_lin, tracer1..."